### DISCUSSION TITLE
Is setting "width" in two-columns layout in PowerPoint really supported?

### DISCUSSION THREAD

**Discussion by mcanouil** [jgm/pandoc#9890] [Q&A]
It's not very clear in which context setting width works or not from <https://pandoc.org/MANUAL.html#columns>.

<table>
<tr><th>Markdown</th><th>PowerPoint</th></tr>
<tr><td>

```sh
pandoc --from markdown --to pptx -o index.pptx index.qmd
```

````qmd
---
title: "Untitled"
---

## Slide title

::: columns
::: {.column width="20%"}
Col 1
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
:::

::: {.column width="80%"}
Col 2
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
:::
:::

````

</td><td>

<img width="1078" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/f95e67f7-876a-4895-b994-73da725bf38a">

</td></tr>
</table>

Is setting "width" in two-columns layout in PowerPoint really supported?

**Comment by jgm** [comment]
I'm not sure.  I take it you tried it and it didn't work?

**Reply by mcanouil** [reply]
It's actually an old report on Quarto CLI that had recent comments.
I tried it today (pandoc 3.2) and it does not work.

Source:

- https://github.com/quarto-dev/quarto-cli/issues/2635

**Reply by mcanouil** [reply]
If that's not supported or supposed to work (I am not sure Pandoc modifies the shape of the slide layout which is what setting width implies), a note to state that (in <https://pandoc.org/MANUAL.html#columns>) would set more appropriate expectations for users.

**Reply by jgm** [reply]
Will do.
